lantern/src/Cuda.cpp

```diff
@@ -4,6 +4,7 @@
 
 #include "lantern/lantern.h"
 
+#include <ATen/cuda/CUDAContext.h>
```

Comment:
You will need to wrap this around something like `#ifdef __CUDACC__` to not include this header when CUDA is not available.

Reply:
Yes, good point. I only tested this on cuda compiled torch previously, and I didn't think about this. This makes perfect sense. Done.